ngraph/core/src/op/rnn_sequence.cpp

```diff
@@ -97,6 +97,20 @@ void op::v5::RNNSequence::validate_and_infer_types() {
                               Dimension::merge(merged_num_directions, merged_num_directions, b_pshape[0]),
                           "Parameter num_directions not matched in RNNSequence.");
 
+    auto valid_num_directions = 0;
```

Comment:
Why do we need to introduce additional restrictions?
It can cause some issues for users who use RNNSequence.

Reply:
I add this restriction because of the explanation in  [RNNSequence_5.md](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/sequence/RNNSequence_5.md)
"Description: Specify if the RNN is forward, reverse, or bidirectional. If it is one of forward or reverse then num_directions = 1, if it is bidirectional, then num_directions = 2. This num_directions value specifies input/output shape requirements."

Reply:
this check is ok